### PR TITLE
CCE: Partial success status codes and documentation

### PR DESCRIPTION
**Pull request by taylorrassmannlibertyits**
https://vajira.max.gov/browse/API-1312
Api 1312 partial success status codes and documentation updated.

### CODE REVIEW

File: community-care-eligibility/src/main/java/gov/va/api/health/communitycareeligibility/service/CommunityCareEligibilityV0ApiController.java [hunk 1]
```diff
@@ -264,6 +264,7 @@ private CommunityCareEligibilityResponse search(PatientRequest request) {
           .eligible(!codeStrings.contains("X"))
           .grandfathered(codeStrings.contains("G"))
           .noFullServiceVaMedicalFacility(codeStrings.contains("N"))
+          .processingStatus(CommunityCareEligibilityResponse.ProcessingStatus.successful)
```
Comment (jblefkove-va) [inline community-care-eligibility/src/main/java/gov/va/api/health/communitycareeligibility/service/CommunityCareEligibilityV0ApiController.java:267]: Would it make sense to set this up on lines 254-259? Then you only need to override it if its not successful.

File: README.md [hunk 2]
```diff
@@ -67,6 +67,13 @@ and scheduled their appointment.
 The response includes the overall eligibility decision and the individual details
 (patient address, eligibility codes, nearby facilities, etc.) that were used to compute it.
 
+It should be noted that partial success states can occur. Partial success states can occur due to:
+```
+Address geocoding is not available
+Address geocoding is out of date compared to the users address
+Address geocoding is incomplete (e.g. missing latitude)
```
Comment (jblefkove-va) [inline README.md:74]: Update the sample response to include the new field

File: community-care-eligibility/src/main/java/gov/va/api/health/communitycareeligibility/service/CommunityCareEligibilityV0ApiController.java (hunk 1 above)
Comment (taylorrassmannlibertyits) [reply community-care-eligibility/src/main/java/gov/va/api/health/communitycareeligibility/service/CommunityCareEligibilityV0ApiController.java:267]: Yep! I'll make the change

**Comment by bschofield-va** [comment]
I did not really review the code, other than the CODEOWNERS so you could move forward at your leisure.

**Pull request merged**